client/src/components/StreamField/blocks/StreamBlock.test.js

```diff
@@ -517,7 +517,8 @@ describe('telepath: wagtail.blocks.StreamBlock with labels that need escaping',
 
   test('it renders correctly', () => {
     boundBlock.inserters[0].open();
-    expect(document.body.innerHTML).toMatchSnapshot();
+    const listbox = document.querySelector('[role="listbox"]');
+    expect(listbox.innerHTML).toMatchSnapshot();
```

Comment:
I’ve changed this so the snapshot is a bit more specific about what we’re testing here (HTML char escaping)